grammar.js

```diff
@@ -260,7 +260,6 @@ module.exports = grammar({
 
     type_qualifier: $ => choice(
       'const',
-      'restrict',
```

Comment:
Did you mean to remove `restrict` here?

Reply:
my bad I should have done this in another commit.
Anyway the keyword is duplicated in the list (just after volatile)

Reply:
Oh, no I didn't realize! That's fine I don't mind them being in the same commit. I usually squash PRs anyway, to avoid extra commits of the generated `parser.c` file.